community/kernel/src/main/java/org/neo4j/kernel/api/explicitindex/AutoIndexOperations.java

```diff
@@ -44,8 +46,11 @@ void propertyChanged( DataWriteOperations ops, long entityId, int propertyKeyId,
     void propertyRemoved( DataWriteOperations ops, long entityId, int propertyKey )
         throws AutoIndexingKernelException;
 
+    //TODO this method should be removed once fully migrated
     void entityRemoved( DataWriteOperations ops, long entityId ) throws AutoIndexingKernelException;
```

Comment:
Does it make sense to deprecate to signal this intention?

Reply:
I have already done the work for cleaning this up in the next chunk of work. Trying to keep PR:s smallish so if you just pretend you didn't see this we'll have it fixed in the next PR.

Reply:
Thank you that effort, of keeping the PRs as small as possible 🎉